_docs_integrate/error-codes.md

```diff
@@ -24,76 +24,167 @@ required_by:
-| {% include anchor a="error.transport.notSupported" %}                                                       | The requested method is not yet supported.                                                                                                                                                                                                                                |
-| {% include anchor a="error.transport.recordNotFound" %}                                                     | The requested record was not found.                                                                                                                                                                                                                                       |
-| {% include anchor a="error.transport.relationships.wrongChangeStatus" %}                                    | The Relationship change has the wrong status to run the requested operation.                                                                                                                                                                                              |
-| {% include anchor a="error.transport.secrets.secretNotFound" %}                                             | No secret was found for a specific type.                                                                                                                                                                                                                                  |
-| {% include anchor a="error.transport.secrets.wrongBaseKeyType" %}                                           | The given secret type is not supported!                                                                                                                                                                                                                                   |
-| {% include anchor a="error.transport.signatureNotValid" %}                                                  | A signature is invalid.                                                                                                                                                                                                                                                   |
-| {% include anchor a="error.transport.tokens.invalidTokenContent" %}                                         | The given token content is invalid.                                                                                                                                                                                                                                       |
+# Error codes regarding the Connector <a href="https://github.com/nmshd/connector"><i class="fab fa-fw fa-github"/></a>
```

Comment:
I don't like the new outline. In general **all** error codes are regarding the connector because they can occur there. I'd suggest to join the tables again.

Reply:
I would keep the separate tables, since they improve readability greatly.

Reply:
The separate tables I have added because the beginning of the error-codes changes and so you can find the error-code faster, but if you want we can change the titles? 
Before I called it like "ErrorCodes with the prefix "error.consumption.attributes." but this wasn't wanted. 
@jkoenig134

Reply:
This table does not have to be readable. All we need is the anchors. I'd still say remove it.